dkg/disk.go

```diff
@@ -33,12 +36,17 @@ import (
 	"github.com/obolnetwork/charon/tbls/tblsconv"
 )
 
-// loadDefinition returns the cluster definition from disk (or the test definition if configured).
+// loadDefinition returns the cluster definition from disk or network. It returns the test definition if configured.
 func loadDefinition(conf Config) (cluster.Definition, error) {
```

Comment:
add an internal test for this function please

Reply:
done. for both `loadDefinition` and `fetchDefinition`